.github/pull_request_template.md

```diff
@@ -19,6 +19,12 @@ Closes #.
 There is no need to duplicate the description in the issue here but it is sometimes worth providing a summary of the individual changes in this PR.
 -->
 
+# How are these changes tested?
+
+<!--
+Was it possible to write an automated test? If not, why?
+-->
+
```

Comment:
```suggestion
# Are these changes tested?

<!--
We typically require tests for all PRs in order to:
1. Prevent the code from being accidentally broken by subsequent changes
2. Serve as another way to document the expected behavior of the code

If tests are not included in your PR, please explain why (for example, are they covered by existing tests)?
-->

```

Reply:
I thought it might make it easier on contributors to say "yes the code is tested" rather than suggesting they may have to write down how it was tested (in addition to the code in the PR)